# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../arpalert/modify_defaultuser.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
#
# Allow arpalert to run under user nobody instead of the unexistent user arpalert
#
--- ./etc/arpalert.conf.in.orig	2006-09-02 00:53:40.000000000 +0300
+++ ./etc/arpalert.conf.in	2006-09-02 17:05:45.000000000 +0300
@@ -12,7 +12,7 @@
 log level = 6
 
 # user for privilege separation
-user = arpalert
+user = nobody
 
 # rights for file creation
 umask = 177

